java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -285,6 +288,24 @@ public void transaction_builds_protobuf_request(BaseTransaction<?> transaction)
         transaction.configResetStat();
         results.add(Pair.of(ConfigResetStat, ArgsArray.newBuilder().build()));
 
+        transaction.configGet(new String[] {"Foster", "The", "Fluffy", "Dogster"});
```

Comment:
what about adding transaction tests?
https://github.com/aws/glide-for-redis/blob/b1ef52b0275a05a2486b71e27f337077ff9248f2/python/python/tests/test_transaction.py#L78